src/components/orgunits/OrgUnitFieldSelect.js

```diff
@@ -34,9 +34,9 @@ const OrgUnitFieldSelect = () => {
     const attribute = attributes.find((a) => a.id === orgUnitField)
 
     return (
-        <>
+        <div className={styles.orgUnitField}>
             <SelectField
-                label={i18n.t('Use associated geometry')}
+                prefix={i18n.t('Use associated geometry')}
                 items={[{ id: NONE, name: i18n.t('None') }, ...attributes]}
                 value={orgUnitField}
```

Comment:
Suggest having None as the default value.

Reply:
Fixed.